src/Integration/RequestIntegration.php

```diff
@@ -41,18 +41,27 @@ final class RequestIntegration implements IntegrationInterface
      */
     private $options;
 
+    /**
+     * Fetches a PSR-7 request object for the current request.
+     *
+     * @var RequestFetcherInterface
+     */
+    private $requestFetcher;
+
     /**
      * Constructor.
      *
-     * @param Options $options The client options
+     * @param Options|null                 $options        The client options
+     * @param RequestFetcherInterface|null $requestFetcher Fetches a PSR-7 request object for the current request
```

Comment:
You can reuse the same description of the property, see above for the reason of this change